apiserver/apiserver.go

```diff
@@ -356,6 +360,17 @@ func newServer(cfg ServerConfig) (_ *Server, err error) {
 
 		healthStatus: "starting",
 	}
+	if refreshURL := controllerConfig.LoginTokenRefreshURL(); refreshURL != "" {
+		if !strings.Contains(refreshURL, "wellknown") {
+			refreshURL = strings.Trim(refreshURL, "/") + "/.well-known/jwks.json"
+		}
+		srv.jwtTokenService = &jwtService{
+			refreshURL: refreshURL,
+		}
+		if err := srv.jwtTokenService.RegisterJWKSCache(context.TODO(), http.DefaultClient); err != nil {
```

Comment:
redirects now disabled, https for the url enforced in config validation but no way to pass in a custom CA cert yet

Reply:
I'll defer the check for https until config is updated to allow a certificate to be configured. This will unblock testing.

Reply:
Btw for testing with https I push a custom CA cert into the lxc container running the controller and restart the container, then https works just fine without configuring the cert in the controller config directly. You can see how this is done in the instructions for the mitm service.

In case you want to add this now, we shouldn't be blocked.